Explorer/Assets/Scripts/Global/Dynamic/MainSceneLoader.cs

```diff
@@ -272,6 +277,27 @@ await bootstrap.InitializeFeatureFlagsAsync(bootstrapContainer.IdentityCache!.Id
             }
         }
 
+        private async UniTask<bool> CheckPlayerBlocked(StaticContainer sc, string playerID, SplashScreen splashScreen, IWebBrowser webBrowser, IDecentralandUrlsSource urlsSource, CancellationToken ct)
```

Comment:
I would rename `CheckPlayerBlocked` to classical `IsUserBlockled`  since it returns bool. Then the `if` statement would be more readable.